### ISSUE TITLE
folder_exclude_patterns not being obeyed for imports

### ISSUE THREAD

**Issue by katanacrimson** [unlight/sublime-import-helper#50]
As on tin. Currently, my project file is the following:

``` json
{
  "SublimeLinter":
  {
    "linters":
    {
      "tslint":
      {
        "args":
        [
          "--config",
          "D:\\code\\redbooru\\api\\tslint.json"
        ]
      }
    }
  },
  "folders":
  [
    {
      "path": "redbooru/api",
      "folder_exclude_patterns": [
        "api/build",
        "api/node_modules",
        "api/temp",
        "api/upload"
      ]
    }
  ]
}
```

It lives at `D:\code\redbooru-api.sublime-project`.  

I have typescript compiling everything from `D:\code\redbooru\api\src\` into `D:\code\redbooru\api\build\`, and have Sublime excluding that directory (and *Sublime* is visually excluding the directory), however when using the plugin, I'm seeing this happen:

![image](https://user-images.githubusercontent.com/121707/35845069-423a91a4-0ad6-11e8-92de-9175b5b4d305.png)

Verifying that we're ignoring the directory properly:

With exclude pattern for `api/build`: 

![image](https://user-images.githubusercontent.com/121707/35845138-988eef6e-0ad6-11e8-9646-8656291a8daf.png)

Without: 

![image](https://user-images.githubusercontent.com/121707/35845144-a2a4569c-0ad6-11e8-9337-a0b6dd5058ce.png)

Doing a cleanout of the `build/` directory and updating imports gives me 125 modules.  Running typescript's compiler again and updating imports gives me 240.

So, yeah, I think we've got a minor problem with obeying `folder_exclude_patterns`. :\

**Comment by unlight** [comment]
As a workaround, you can change pattern to 
```
  "folder_exclude_patterns": [
    "redbooru/api/build",
    "redbooru/api/node_modules",
    "redbooru/api/temp",
    "redbooru/api/upload"
  ]
```

I'll fix it in next release.

**Comment by katanacrimson** [comment]
Cool - workaround works fine. Thanks much!

**Comment by unlight** [comment]
Fixed by 9387ba3867a936a8a00a199b8898bded7d1c3593 will be shipped to v1.8.1
